assert.js

```diff
@@ -83,11 +86,24 @@ assertplus.response = function(req, res, callback) {
                             '    Got: ' + util.inspect(response.body)
                     );
                 }
+                // handle gzipped responses
+                if (response.headers['content-encoding'] == 'gzip') {
+                    zlib.gunzip(response.body, function(err, buffer){
+                        if (err) { return callback(err);
```

Comment:
Code-style wise, seems like `return callback(err);` should be on the next line here?